code/script.js

```diff
@@ -30,18 +35,89 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Bot Starts talking here
 const greeting = () => {
   showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  
 }
+//This first setTimeout 
+setTimeout(greeting, 1000)
 
-// Set up your eventlisteners here
+//here is the information on what name you have 
+const handleInput = (event) => {
+  event.preventDefault()
+  const inputValue = document.getElementById('name-input').value
+  showMessage(inputValue, 'user')
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+//and here it will be displayed by the bot🤖
+  setTimeout(function() {
+    showMessage(`Hello ${inputValue}, nice name 🌞`, 'bot')
+    form.style.display="none"}, 1000)
+  
+
+  setTimeout(function() {
+    showMessage(`What cat fact do you want today?`, 'bot')
+    factButtons.style.display="flex"}, 2000)
+}
+//Here we have our facts buttons. After we are more comfortable with JS it would be fun putting these buttons all in one conditional statement!
+const factOne = () => {
+  factButtons.style.display="none"
+  setTimeout(function() {
+    showMessage(`Fact 1`, 'user')}, 1000)
+  setTimeout(function() {
+    showMessage(`Cat whiskers are the same width as their body😸`, 'bot')
+    }, 2000)
+  setTimeout(() => showMessage(`Do you want another epic cat fact?`, 'bot'), 3000)
+  setTimeout(() => yesNoButton.style.display="flex", 4000) 
+}
+
+const factTwo = () => {
+  factButtons.style.display="none"
+  setTimeout(function() {
+    showMessage(`Fact 2`, 'user')}, 1000)
+  setTimeout(function() {
+    showMessage(`While us humans have 206 bones, cats on average have 244. It ranges between 230-250 depending on how long a cat’s tail is and how many toes the cat has🐈`, 'bot')}, 2000)
+  setTimeout(() => showMessage(`Do you want another epic cat fact?`, 'bot'), 3000)
+  setTimeout(() => yesNoButton.style.display="flex", 4000) 
```

Comment:
Here is an other place where the chat scroll can be fixed like the comment on line 60